ARM no longer building with glibc-2.30

`glibc-2.30` removes [sys/io.h](https://sourceware.org/glibc/wiki/Release/2.30#A.3Csys.2BAC8-io.h.3E_removed_on_32-bit_Arm) for ARM, and `lm_sensors-3.5.0` (for arch `arm`) no longer builds.

ARM build log: http://ix.io/1Qq2

```
sed -e 's@^\(.*\)\.o:@lib/conf-parse.ld lib/conf-parse.lo: Makefile '`dirname lib/conf-parse.ld`/Module.mk' @' > lib/conf-parse.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/conf-lex.c | \
sed -e 's@^\(.*\)\.o:@lib/conf-lex.ad lib/conf-lex.ao: Makefile '`dirname lib/conf-lex.ad`/Module.mk' @' > lib/conf-lex.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/conf-lex.c | \
sed -e 's@^\(.*\)\.o:@lib/conf-lex.ld lib/conf-lex.lo: Makefile '`dirname lib/conf-lex.ld`/Module.mk' @' > lib/conf-lex.ld
make: *** No rule to make target 'sys/io.h', needed by 'prog/dump/superio.rd'.  Stop.
```

> Can you include a link to the build log for completeness?

Building for RPi (armv6) with this PR:
```
[1;36mUNPACK[0m      lm_sensors
    [1;32mAPPLY PATCH [1;37m(common)[0m[0m      packages/addons/addon-depends/system-tools-depends/lm_sensors/patches/lm_sensors-01_link-static.patch
patching file prog/sensors/Module.mk
    [1;32mAPPLY PATCH [1;37m(common)[0m[0m      packages/addons/addon-depends/system-tools-depends/lm_sensors/patches/lm_sensors-02-PR192.patch
patching file Makefile
[1;33mBUILD[0m      lm_sensors [1;37m(target)[0m
    [1;35mTOOLCHAIN[0m      make (auto-detect)
Executing (target): make PREFIX=/usr CC=/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc AR=/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-ar
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 prog/sensors/chips.c | \
sed -e 's@^\(.*\)\.o:@prog/sensors/chips.rd prog/sensors/chips.ro: Makefile '`dirname prog/sensors/chips.rd`/Module.mk' @' > prog/sensors/chips.rd
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 prog/sensors/main.c | \
sed -e 's@^\(.*\)\.o:@prog/sensors/main.rd prog/sensors/main.ro: Makefile '`dirname prog/sensors/main.rd`/Module.mk' @' > prog/sensors/main.rd
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/sysfs.c | \
sed -e 's@^\(.*\)\.o:@lib/sysfs.ad lib/sysfs.ao: Makefile '`dirname lib/sysfs.ad`/Module.mk' @' > lib/sysfs.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/init.c | \
sed -e 's@^\(.*\)\.o:@lib/init.ad lib/init.ao: Makefile '`dirname lib/init.ad`/Module.mk' @' > lib/init.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/bison
bison -p sensors_yy -d lib/conf-parse.y -o lib/conf-parse.c
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/access.c | \
sed -e 's@^\(.*\)\.o:@lib/access.ad lib/access.ao: Makefile '`dirname lib/access.ad`/Module.mk' @' > lib/access.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/error.c | \
sed -e 's@^\(.*\)\.o:@lib/error.ad lib/error.ao: Makefile '`dirname lib/error.ad`/Module.mk' @' > lib/error.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/general.c | \
sed -e 's@^\(.*\)\.o:@lib/general.ad lib/general.ao: Makefile '`dirname lib/general.ad`/Module.mk' @' > lib/general.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/data.c | \
sed -e 's@^\(.*\)\.o:@lib/data.ad lib/data.ao: Makefile '`dirname lib/data.ad`/Module.mk' @' > lib/data.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/sysfs.c | \
sed -e 's@^\(.*\)\.o:@lib/sysfs.ld lib/sysfs.lo: Makefile '`dirname lib/sysfs.ld`/Module.mk' @' > lib/sysfs.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/init.c | \
sed -e 's@^\(.*\)\.o:@lib/init.ld lib/init.lo: Makefile '`dirname lib/init.ld`/Module.mk' @' > lib/init.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/access.c | \
sed -e 's@^\(.*\)\.o:@lib/access.ld lib/access.lo: Makefile '`dirname lib/access.ld`/Module.mk' @' > lib/access.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/error.c | \
sed -e 's@^\(.*\)\.o:@lib/error.ld lib/error.lo: Makefile '`dirname lib/error.ld`/Module.mk' @' > lib/error.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/general.c | \
sed -e 's@^\(.*\)\.o:@lib/general.ld lib/general.lo: Makefile '`dirname lib/general.ld`/Module.mk' @' > lib/general.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/data.c | \
sed -e 's@^\(.*\)\.o:@lib/data.ld lib/data.lo: Makefile '`dirname lib/data.ld`/Module.mk' @' > lib/data.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/conf-parse.c | \
sed -e 's@^\(.*\)\.o:@lib/conf-parse.ad lib/conf-parse.ao: Makefile '`dirname lib/conf-parse.ad`/Module.mk' @' > lib/conf-parse.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/conf-parse.c | \
sed -e 's@^\(.*\)\.o:@lib/conf-parse.ld lib/conf-parse.lo: Makefile '`dirname lib/conf-parse.ld`/Module.mk' @' > lib/conf-parse.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/flex
flex -Psensors_yy -t -Cfe -8 lib/conf-lex.l > lib/conf-lex.c
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/conf-lex.c | \
sed -e 's@^\(.*\)\.o:@lib/conf-lex.ad lib/conf-lex.ao: Makefile '`dirname lib/conf-lex.ad`/Module.mk' @' > lib/conf-lex.ad
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -M -MG -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 lib/conf-lex.c | \
sed -e 's@^\(.*\)\.o:@lib/conf-lex.ld lib/conf-lex.lo: Makefile '`dirname lib/conf-lex.ld`/Module.mk' @' > lib/conf-lex.ld
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/data.c -o lib/data.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/general.c -o lib/general.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/error.c -o lib/error.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/access.c -o lib/access.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/init.c -o lib/init.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/sysfs.c -o lib/sysfs.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -Wno-undef -c lib/conf-parse.c -o lib/conf-parse.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -fpic -D_REENTRANT -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -Wno-shadow -Wno-undef -Wno-unused -Wno-missing-prototypes -Wno-sign-compare -c lib/conf-lex.c -o lib/conf-lex.lo
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/data.c -o lib/data.ao
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/general.c -o lib/general.ao
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/error.c -o lib/error.ao
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/access.c -o lib/access.ao
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/init.c -o lib/init.ao
In file included from /home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/armv6zk-libreelec-linux-gnueabi/sysroot/usr/include/sys/types.h:25,
                 from lib/init.c:25:
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/armv6zk-libreelec-linux-gnueabi/sysroot/usr/include/features.h:185:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^~~~~~~
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/sysfs.c -o lib/sysfs.ao
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -Wno-undef -c lib/conf-parse.c -o lib/conf-parse.ao
lib/sysfs.c: In function 'find_bus_type':
lib/sysfs.c:637:8: warning: '/class/i2c-adapter/i2c-' directive output may be truncated writing 23 bytes into a region of size between 1 and 255 [-Wformat-truncation=]
     "%s/class/i2c-adapter/i2c-%d/device",
        ^~~~~~~~~~~~~~~~~~~~~~~
lib/sysfs.c:637:5: note: directive argument in the range [-32768, 32767]
     "%s/class/i2c-adapter/i2c-%d/device",
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lib/sysfs.c:636:4: note: 'snprintf' output between 32 and 291 bytes into a destination of size 255
    snprintf(bus_path, sizeof(bus_path),
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     "%s/class/i2c-adapter/i2c-%d/device",
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     sensors_sysfs_mount, entry->chip.bus.nr);
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lib/sysfs.c: In function 'sensors_read_dynamic_chip':
lib/sysfs.c:408:31: warning: '%s' directive output may be truncated writing up to 255 bytes into a region of size 254 [-Wformat-truncation=]
  snprintf(path, NAME_MAX, "%s/%s", device, attr);
                               ^~
lib/sysfs.c:408:2: note: 'snprintf' output 2 or more bytes (assuming 257) into a destination of size 255
  snprintf(path, NAME_MAX, "%s/%s", device, attr);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -Wno-shadow -Wno-undef -Wno-unused -Wno-missing-prototypes -Wno-sign-compare -c lib/conf-lex.c -o lib/conf-lex.ao
In file included from /home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/armv6zk-libreelec-linux-gnueabi/sysroot/usr/include/sys/types.h:25,
                 from lib/init.c:25:
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/armv6zk-libreelec-linux-gnueabi/sysroot/usr/include/features.h:185:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^~~~~~~
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -shared -march=armv6zk -mtune=arm1176jzf-s -Wl,--as-needed -fuse-ld=gold -Wl,--version-script=lib/libsensors.map -Wl,-soname,libsensors.so.5 -o lib/libsensors.so.5.0.0 lib/data.lo lib/general.lo lib/error.lo lib/access.lo lib/init.lo lib/sysfs.lo lib/conf-parse.lo lib/conf-lex.lo -lc -lm
rm -f lib/libsensors.so.5
rm -f lib/libsensors.so 
ln -sf libsensors.so.5.0.0 lib/libsensors.so.5
ln -sf libsensors.so.5.0.0 lib/libsensors.so
lib/sysfs.c: In function 'find_bus_type':
lib/sysfs.c:637:8: warning: '/class/i2c-adapter/i2c-' directive output may be truncated writing 23 bytes into a region of size between 1 and 255 [-Wformat-truncation=]
     "%s/class/i2c-adapter/i2c-%d/device",
        ^~~~~~~~~~~~~~~~~~~~~~~
lib/sysfs.c:637:5: note: directive argument in the range [-32768, 32767]
     "%s/class/i2c-adapter/i2c-%d/device",
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lib/sysfs.c:636:4: note: 'snprintf' output between 32 and 291 bytes into a destination of size 255
    snprintf(bus_path, sizeof(bus_path),
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     "%s/class/i2c-adapter/i2c-%d/device",
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     sensors_sysfs_mount, entry->chip.bus.nr);
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lib/sysfs.c: In function 'sensors_read_dynamic_chip':
lib/sysfs.c:408:31: warning: '%s' directive output may be truncated writing up to 255 bytes into a region of size 254 [-Wformat-truncation=]
  snprintf(path, NAME_MAX, "%s/%s", device, attr);
                               ^~
lib/sysfs.c:408:2: note: 'snprintf' output 2 or more bytes (assuming 257) into a destination of size 255
  snprintf(path, NAME_MAX, "%s/%s", device, attr);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rm -f lib/libsensors.a
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-ar rcvs lib/libsensors.a lib/data.ao lib/general.ao lib/error.ao lib/access.ao lib/init.ao lib/sysfs.ao lib/conf-parse.ao lib/conf-lex.ao
a - lib/data.ao
a - lib/general.ao
a - lib/error.ao
a - lib/access.ao
a - lib/init.ao
a - lib/sysfs.ao
a - lib/conf-parse.ao
a - lib/conf-lex.ao
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c prog/sensors/main.c -o prog/sensors/main.ro
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c prog/sensors/chips.c -o prog/sensors/chips.ro
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -Wl,-rpath,/usr/lib -o prog/sensors/sensors prog/sensors/main.ro prog/sensors/chips.ro  -Llib -Wl,-Bstatic -lsensors -Wl,-Bdynamic -lm
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -DETCDIR="\"/etc\"" -I.  -Wall -O2 -Wstrict-prototypes -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wnested-externs -Winline -W -Wmissing-prototypes -Wundef -march=armv6zk -mtune=arm1176jzf-s -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=vfp -fomit-frame-pointer -Wall -pipe  -O2 -c lib/test/test-scanner.c -o lib/test/test-scanner.ro
/home/ubuntu/projects/LibreELEC.tv/build.LibreELEC-RPi.arm-9.80-devel/toolchain/bin/armv6zk-libreelec-linux-gnueabi-gcc -Wl,-rpath,/usr/lib -o lib/test/test-scanner lib/test/test-scanner.ro lib/conf-lex.ao lib/error.ao lib/general.ao -Llib
```

--- a/Makefile
+++ b/Makefile
@@ -77,7 +77,7 @@ LIBINCLUDEDIR := $(INCLUDEDIR)/sensors
 # manual pages will be installed.
 MANDIR := $(PREFIX)/man
 
-MACHINE := $(shell uname -m)
+ARCH := $(firstword $(subst -, ,$(shell $(CC) -dumpmachine)))
 
 # Extra non-default programs to build; e.g., sensord
 #PROG_EXTRA := sensord
@@ -109,7 +109,7 @@ BUILD_STATIC_LIB := 1
 SRCDIRS := lib prog/detect prog/pwm \
            prog/sensors ${PROG_EXTRA:%=prog/%} etc
 # Only build isadump and isaset on x86 machines.
-ifneq (,$(findstring $(MACHINE), i386 i486 i586 i686 x86_64))
+ifneq (,$(findstring $(ARCH), i386 i486 i586 i686 x86_64))
 SRCDIRS += prog/dump
 endif
 SRCDIRS += lib/test

